src/BTConnectionManager.cpp

```diff
@@ -134,6 +143,23 @@ void BTConnectionManager::setAppSettings(AppSettings* appSettings)
     d->appSettings = appSettings;
 }
 
+void BTConnectionManager::setLocalBTDeviceState()
+{   //0-off, 1-on, 2-no device
+    //TODO: use enum?
```

Comment:
Nice idea, certainly would be a nice bit of API design (and also in line with the Qt guidelines) :)